src/libServer/Server.cpp

```diff
@@ -147,9 +144,7 @@ bool Server::StartCollectorThread() {
         continue;
       }
 
-      LOG_GENERAL(INFO, "Sent to " << upperLayerNode);
-
-      P2PComm::GetInstance().SendMessage(upperLayerNode, msg);
+      m_mediator.m_lookup->SendMessageToRandomSeedNode(msg);
```

Comment:
I think we can just remove the func `GetAboveLayer`

Reply:
Done